torch/_functorch/partitioners.py

```diff
@@ -204,7 +204,7 @@ def _extract_graph_with_inputs_outputs(
             output_values.append(env[x])
         else:
             output_values.append(x)
-    new_graph.output(output_values)
+    new_graph.output(tuple(output_values))
```

Comment:
cc @Chillee - I think this should make the graph a bit more consists between inference and training - previously the inference graphs we generated would return tuple outputs, while the fw and bw graphs that we get from the partitioner would return list outputs